### DISCUSSION TITLE
⛔ Add Shortcode as a Dynamic Tag option for Images

### DISCUSSION THREAD

**Discussion by desidm** [elementor/elementor#18634] [Feature Request]
Hi, Image widget - Add shortcode option for dynamic tags. I need to grab an image from somewhere outside the current post so using the ACF image field is not enough.

**Comment by nicholaszein** [comment]
Hello @desidm!

We cannot offer this functionality because we can't predict what a shortcode would output, making it impossible to prevent possible issues caused by any code that could be injected in the `src` attribute of `<img>` tags. This also poses a security risk.

You can, however, try to create your custom dynamic tag for your purpose by following the [Developers Documentation](https://developers.elementor.com/docs/dynamic-tags/).

Thank you for understanding! 🙏 

Best of luck!
